frontend/src/pages/users/Users.jsx

```diff
@@ -95,6 +96,15 @@ const Users = () => {
       align: "center",
       render: (forms) => forms.length || "None",
     },
+    {
+      title: "Last Login",
+      dataIndex: "last_login",
+      align: "center",
+      render: (last_login) =>
+        last_login
+          ? moment.unix(last_login).format("MMMM Do YYYY, h:mm:ss a")
```

Comment:
@wayangalihpratama, second is not necessary be here. Please merge to develop yourself once you change this.